modules/persistent-storage-local-install.adoc

```diff
@@ -34,6 +34,17 @@ To block the Local Storage Operator from inheriting the cluster-wide default sel
 $ oc annotate project openshift-local-storage openshift.io/node-selector=''
 ----
 
+. Optional: Allow local storage to run on the management pool of CPUs in single-node deployment.
++
+Use the Local Storage Operator in single-node deployments and allow the use of CPUs that belong to the `managment` pool. Perform this step on single-node installations that use management workload partitioning.
++
+To allow Local Storage Operator to run on the management CPU pool, run following command:
++
+[source,terminal]
+----
+$ oc annotate project workload.openshift.io/allowed='management'
```

Comment:
1. Missing the name "openshift-local-storage"
2. Looks like "oc annotate project" doesn't work: 
```
$ oc annotate project openshift-local-storage openshift.io/node-selector=''
The Project "openshift-local-storage" is invalid: 
* metadata.namespace: Invalid value: "openshift-local-storage": field is immutable
* metadata.namespace: Forbidden: not allowed on this type

$ oc annotate project openshift-local-storage workload.openshift.io/allowed='management'
The Project "openshift-local-storage" is invalid: 
* metadata.namespace: Invalid value: "openshift-local-storage": field is immutable
* metadata.namespace: Forbidden: not allowed on this type
* metadata.annotations[workload.openshift.io/allowed]: Invalid value: "management": field is immutable, try updating the namespace
```

3. Please use "oc annotate namespace" instead. (Including another cmd not from this PR)
```
 $ oc annotate namespace openshift-local-storage openshift.io/node-selector=''
namespace/openshift-local-storage annotated

$ oc annotate namespace openshift-local-storage workload.openshift.io/allowed='management'
namespace/openshift-local-storage annotated
```